pulsar-common/src/main/proto/PulsarMarkers.proto

```diff
@@ -80,10 +80,12 @@ message ClusterMessageId {
 message MessageIdData {
 	required uint64 ledger_id = 1;
 	required uint64 entry_id  = 2;
+    optional int32 partition = 3 [default = -1];
```

Comment:
Why we need partition information in the marker?  A marker can only belong to a single partition right?

Reply:
Yes, I'll fix this.